WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/editor/WatchFaceConfigActivity.kt

```diff
@@ -123,6 +123,16 @@ class WatchFaceConfigActivity : ComponentActivity() {
         stateHolder.setColorStyle(newColorStyle.id)
     }
 
+    fun onClickLeftComplicationButton(view: View) {
+        Log.d(TAG, "onClickLeftComplicationButton() $view")
+        stateHolder.setComplication(LEFT_COMPLICATION_ID)
+    }
+
+    fun onClickRightComplicationButton(view: View) {
+        Log.d(TAG, "onClickLeftComplicationButton() $view")
```

Comment:
Small nit (assuming these logs will be removed once the issue is cleared up), but this one also says `onClickLeft` instead of `onClickRight`

Reply:
Fixed.